content/en/docs/apps/yaml-definition.md

```diff
@@ -32,23 +32,23 @@ and set up are defined inside the application YAML files. Darcy Cloud allows for
+| `container.ports`          | List of port mapping to be provided to the container running the microservice. See the [Docker to Darcy]({{<ref "docker-to-darcy.md">}}) section for a more detailed explanation about this field.                                                                                                                                                                                                                                                                                                    |
+| `container.ports.external` | The external port exposed to the node.                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
+| `container.ports.internal` | The internal port used by the microservice.                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
+| `container.ports.protocol` | The protocol used to send data between the internal and external ports via a proxy tunnel. The choices for the field are `tcp` or `udp`.                                                                                                                                                                                                                                                                                                                                               |
+| `container.ports.scheme` | Define the protocol to be used when accessing the proxy.                                                                                                                                                                                                                                                                                                               || `container.ports.public`                 | This is a specialized field that exposes a `public` port to the internet at large. Setting this port allows the URL to map directly to the internal port outside of the microservice. See [public services]({{<ref "port-broker.md">}}) for a more detailed explanation of the public section.                                                                                                                                                                                                    |
```

Comment:
This row seem to be malformed. It shows `scheme` and  `volumes` is after. `container.ports.proxy` must be after `scheme`. Also, make sure to change `public` to `proxy`.